Detectors/ITSMFT/ITS/tracking/src/VertexerTraits.cxx

```diff
@@ -143,15 +143,17 @@ void VertexerTraits::computeTracklets()
       mVrtParams.phiCut,
       mTimeFrame->getTracklets()[0],
       mTimeFrame->getNTrackletsCluster(rofId, 0),
-      mIndexTableUtils);
+      mIndexTableUtils,
+      mVrtParams.maxTrackletsPerCluster);
     trackleterKernelSerial<TrackletMode::Layer1Layer2>(
       mTimeFrame->getClustersOnLayer(rofId, 2),
       mTimeFrame->getClustersOnLayer(rofId, 1),
       mTimeFrame->getIndexTable(rofId, 2).data(),
       mVrtParams.phiCut,
       mTimeFrame->getTracklets()[1],
       mTimeFrame->getNTrackletsCluster(rofId, 1),
-      mIndexTableUtils);
+      mIndexTableUtils,
+      mVrtParams.maxTrackletsPerCluster);
```

Comment:
Thanks, yes, but should not you substitute https://github.com/AliceO2Group/AliceO2/blob/b8c5a0925f225d26a80aabed0cd68d95a3ce95cb/Detectors/ITSMFT/ITS/tracking/src/VertexerTraits.cxx#L45 by the `mVrtParams.maxTrackletsPerCluster` ?

BTW, is 2e3 tracklets/cluster based on some study? Looks quite high.

Reply:
> Thanks, yes, but should not you substitute
> 
> https://github.com/AliceO2Group/AliceO2/blob/b8c5a0925f225d26a80aabed0cd68d95a3ce95cb/Detectors/ITSMFT/ITS/tracking/src/VertexerTraits.cxx#L45
> 
> by the `mVrtParams.maxTrackletsPerCluster` ?
Isn't it done at line 156, or I missed the point?

> BTW, is 2e3 tracklets/cluster based on some study? Looks quite high.
It is high indeed on purpose as the number actually comes from some super-safe assumption done in case of limited memory (gpu case).
There the plan was to have a hard limit but to keep all the tracklets.

If we now want to have a reasonable default indeed this must be studied/estimated.

Also this number is quite obsolete as even the GPU approach now does not afford anymore so many tracklets (due to the rof->tf change of approach)

Reply:
> Isn't it done at line 156, or am I missing the point?

Right, sorry, confused it with local contant.